pod/perlfunc.pod

```diff
@@ -6059,6 +6060,37 @@ X<__PACKAGE__>
 
 A special token that returns the name of the package in which it occurs.
 
+=item __CLASS__
+X<__CLASS__>
+
+=for Pod::Functions the class name of the current instance.
+
+Invoked within a C<method> or similar location, such as a field initializer
```

Comment:
I think `method` and `__PACKAGE__` should be linkified.

Reply:
Added; forcepushed.